gulpfile.js

```diff
@@ -83,10 +89,14 @@ function getRunDebugAppCommand() {
 
         break;
     case 'win32':
-        return path.join(debugDir, pkg.name, 'win32', pkg.name + '.exe');
+		if (os.arch() == 'x64') {
```

Comment:
Same here, add a separate `case 'win64':` to split code paths for `win32` and `win64`.

Reply:
This method tries to execute the program that belongs to your actual platform. Is an independent method, ignores the parameters passed in the command line. Maybe is a bug, but this PR is not to solve this bug.